p2p/host/resource-manager/limit_defaults.go

```diff
@@ -492,8 +492,8 @@ var DefaultLimits = ScalingLimitConfig{
 	},
 
 	PeerBaseLimit: BaseLimit{
-		ConnsInbound:    4,
-		ConnsOutbound:   8,
+		ConnsInbound:    6,
+		ConnsOutbound:   6,
```

Comment:
Why are you modifying the limits here?

Reply:
A bit more info in the commit: https://github.com/libp2p/go-libp2p/commit/a969c6d41aa21a18901a5d45954bc91560c8a53c

We see flakiness in CI because a client will dial with concurrency 8 but the limit on the server is 4 so we get rcmgr failures. We notice this when we have a lot of addrs to dial